webview-ui/src/Periscope/ErrorView.tsx

```diff
@@ -0,0 +1,39 @@
+import { VSCodeDivider } from "@vscode/webview-ui-toolkit/react";
+import { PeriscopeTypes } from "../../../src/webview-contract/webviewTypes";
+import styles from "./Periscope.module.css";
+
+export interface ErrorViewProps {
+    clusterName: string
+    message: string
+    config: PeriscopeTypes.KustomizeConfig
+}
+
+export function ErrorView(props: ErrorViewProps) {
+    return (
+        <>
+            <i className={[styles.errorIndicator, "fa", "fa-times-circle"].join(" ")}></i>
+            AKS Periscope failed to run on '{props.clusterName}'. Please see the error below for more details.
+
+            <VSCodeDivider />
```

Comment:
The webviews used to contain collapsible panels. I was too lazy to try and reproduce those, and I'm not sure if people actually use that functionality (I certainly never did). So I opted for a simple divider, which is part of the UI toolkit and looks reasonably clean.